Test/ElasticScale.ShardManagement.UnitTests/RecoveryManagerTests.cs

```diff
@@ -806,19 +734,19 @@ public void TestCopyGSMToLSM()
             // Briefly validate that there are, in fact, the two ranges of inconsistency we are expecting.
             IEnumerable<RecoveryToken> gs = rm.DetectMappingDifferences(sl);
 
-            Assert.AreEqual(1, gs.Count(), "The test environment was not expecting more than one local shardmap.");
+            Assert.True(1 == gs.Count(), "The test environment was not expecting more than one local shardmap.");
```

Comment:
I will stop commenting about Assert.Equal, hopefully these can be found and replaced by regex :)